klippy/extras/load_cell.py

```diff
@@ -0,0 +1,38 @@
+# Load Cell Implementation
+#
+# Copyright (C) 2024 Gareth Farrington <gareth@waves.ky>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+from . import hx71x
+from . import ads1220
+
+# Printer class that controls a load cell
+class LoadCell:
+    def __init__(self, config, sensor):
+        self.printer = printer = config.get_printer()
+        self.sensor = sensor   # must implement BulkAdcSensor
+        # startup, when klippy is ready, start capturing data
+        printer.register_event_handler("klippy:ready", self._handle_ready)
+
+    def _handle_ready(self):
+        self.sensor.add_client(self._on_sample)
+
+    def _on_sample(self, msg):
+        return True
```

Comment:
Why does this code start sampling at startup?

Reply:
In the next PR the load cell will report the current weight continuously as part of its status, similar to a temperature sensor. This is the smallest part of that I could think if leaving it to get something that actually runs the sensors.

Reply:
For testing, you can use the motan interface - something like: `~/klipper/scripts/motan/data_logger.py /tmp/klippy_uds /tmp/mytestdata`

You may need to modify `scripts/motan/data_logger.py` to enable queries by default.  (Or, otherwise use `scripts/whconsole.py` and manually issue the subscribe requests.)

As part of the initial commit, we do want some minimal functionality - I think the web interface is sufficient for that requirement.  I think we should avoid "debugging code" as part of that first commit.